### DISCUSSION TITLE
현재 Github Convert to issue 기능에 관한 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#57] [Q&A] [answered]
기능이 막힌게 맞나요...?

구글링해도 안나오고 질문글도 올렸는데 혹시나 아실까 해서 연락드립니다..!

https://github.com/orgs/community/discussions/52030

**Comment by jongfeel** [comment, marked as answer]
저도 몰랐는데 안되네요. 뭔가 기능을 빠뜨리고 업데이트 한게 아닐까 싶습니다.

**Reply by fkdl0048** [reply]
으악 ㅠㅠㅠ...!

**Reply by jongfeel** [reply]
github 애들이 잠수함 패치를 했는지 제가 지금 다시 해보니 되는 것 같습니다.
